pokemonterminal/slideshow.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3.6
+import atexit
+import random
+import sys
+
+from threading import Thread
+
+def __print_fork(pid, length, delay):
+    print(f"Starting slideshow with {length} Pokemons and a delay of {delay} minutes.")
+    print(f"Forked process to background with PID {pid}. You can stop it with 'pokemon -c'.")
+
+def __exit_listener():
+    raise Exception('TODO: listen for exit')
+
+def __slideshow_worker(filtered, delay, changer_func):
+    t = Thread(target=__exit_listener)
```

Comment:
According to the threading docs:
```
CPython implementation detail: In CPython, due to the Global Interpreter Lock, only one thread can execute Python code at once (even though certain performance-oriented libraries might overcome this limitation). If you want your application to make better use of the computational resources of multi-core machines, you are advised to use multiprocessing. However, threading is still an appropriate model if you want to run multiple I/O-bound tasks simultaneously.
```

However since we are not using multithreading for anything performance critical, we should be able to get away with this fine. It's also possible I decide to make multi instance use mutexes, which means that thread would literally be sleeping until someone sets the mutex.